instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -172,10 +172,10 @@ def client_response_hook(span: Span, scope: dict[str, Any], message: dict[str, A
 ---
 """
 import logging
-from importlib.util import find_spec
 from typing import Collection
 
 import fastapi
+from pkg_resources import get_distribution
```

Comment:
Does `pkg_resources` come with the standard Python distribution or do we need to add a direct dependency for `setuptools` which includes `pkg_resources`?